producers/smd_producer.py

```diff
@@ -85,7 +114,7 @@ def getROIs(run):
 
 # DEFINE DETECTOR AND ADD ANALYSIS FUNCTIONS
 def define_dets(run):
-    detnames = ['jungfrau1M'] # add detector here
+    detnames = ['jungfrau1M', 'Rayonix', 'epix10k2M'] # add detector here
```

Comment:
I'm not convinced we want to start to list _all_ possible detectors here. Which one or two is/are present, I don't care much about.

Reply:
Agreed. This is a part that should be updated for each experiment anyway.